app/models/recommendation.rb

```diff
@@ -0,0 +1,4 @@
+class Recommendation < ActiveRecord::Base
+  belongs_to :recommendable, polymorphic: true
```

Comment:
> Do you think it'll be tougher to change later on?
 
Yeah, that was my concern. Want me to switch back to a basic `has_many :through` and we'll see how it shakes out?